charts/stable/traefik/templates/middlewares/theme-park.yaml

```diff
@@ -3,18 +3,20 @@
 {{- if or ( not .Values.ingressClass.enabled ) ( and ( .Values.ingressClass.enabled ) ( .Values.ingressClass.isDefaultClass ) ) }}
 {{- $namespace = "default" }}
 {{- end }}
-{{- range $index, $middlewareData := .Values.middlewares.redirectScheme }}
+{{- range $index, $middlewareData := .Values.middlewares.themePark }}
 
 ---
-# Declaring the user list
 apiVersion: traefik.containo.us/v1alpha1
 kind: Middleware
 metadata:
   name: {{ $middlewareData.name }}
   namespace: {{ $namespace }}
 spec:
-    plugin:
-        traefik-themepark:
-            app: {{ $middlewareData.app }}
-            theme: {{ $middlewareData.theme }}
-{{- end }}
+  plugin:
+    traefik-themepark:
+      app: {{ $middlewareData.appName }}
+      theme: {{ $middlewareData.themeName }}
+      {{- if $middlewareData.enableBaseUrl }}
```

Comment:
```suggestion
      {{- if $middlewareData.enableBaseUrl -}}
```

Reply:
I think this causes an issue with the baseUrl: reference. I will test to validate

Reply:
can confirm that whitespace delete breaks the formatting of baseUrl reference.